{{cookiecutter.project_slug}}/src/data/dataGenerator.js

```diff
@@ -0,0 +1,45 @@
+import _ from 'lodash';
+import users from './raw/users';
+import articles from './raw/articles';
+import notifications from './raw/notifications';
+import conversations from './raw/conversations';
+
+const populateArticles = () => {
+  articles.forEach(article => {
+    const userArticle = article;
+    const userId = articles.indexOf(article) % users.length;
+    userArticle.user = _.find(users, x => x.id === userId) || users[0];
```

Comment:
why not just use `find` from ES6?

Reply:
both works

Reply:
I know it works, I tested my self, find from `ES6` is native and faster to create JS build

Reply:
let's have a new PR to remove lodash